### PR TITLE
Coordinates lat lon service

### PR DESCRIPTION
**Pull request by shreshtha-pankaj**

### CODE REVIEW

**Comment by shreshtha-pankaj** [comment]
Two methods have been provided.
1. Get the LAT, LON of a city, country pair. @Returns - if found returns a dict of LAT, LON as key, otherwise an empty dict.
2. For a given city, it also returns the boundary of the city.

File: rest-twitter/service/Location_Service.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                readcsv = csv.reader(file, delimiter=CSV_DELIM)
+                for _line in readcsv:
+                    # location,country_code,lat,long
+                    _key = LocationCountryPair(_line[0].strip(), _line[1].strip())
+                    _val = {LAT: float(_line[2]), LON: float(_line[3])}
+                    self.cache[_key] = _val
+        except FileNotFoundError:
+            print("Load file. File not found: " + file_location)
+
+    def getCordinatesforcity(self, query):
```
Comment (skinnerr) [inline rest-twitter/service/Location_Service.py]: refactor with underscores and spelling of "coordinates"
```suggestion
    def get_coordinates_for_city(self, query):
```

File: rest-twitter/service/Location_Service.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                    return lat_lon
+                else:
+                    print("Can not find lat lon for query:" + query)
+                    return lat_lon
+
+            except Exception as ex:
+                print(ex)
+                return {}
+
+    def getcityboundry(self, query):
```
Comment (skinnerr) [inline rest-twitter/service/Location_Service.py]: refactor
```suggestion
    def get_city_boundary(self, query):
```

**Review by skinnerr — CHANGES_REQUESTED** [review]
Looks good, just some small changes with naming of methods. For readability, it would be beneficial to add docstrings to each class/method.

**Review by skinnerr — APPROVED** [review]

**Pull request merged**